### PR TITLE
changes to generalize get_bounds

### PR DESCRIPTION
**Pull request by martham93**
to address issue #169 

cc @drewbo

### CODE REVIEW

File: label_maker/main.py [hunk 1]
```diff
@@ -20,9 +23,14 @@
...
 
 def get_bounds(feature_collection):
-    """Get a bounding box for a FeatureCollection of Polygon Features"""
-    features = [f for f in feature_collection['features'] if f['geometry']['type'] in ['Polygon']]
-    return MultiPolygon(list(map(lambda x: Polygon(x['geometry']['coordinates'][0]), features))).bounds
+    """Get a bounding box for a FeatureCollection"""
+    features = [f for f in feature_collection['features']]
+    shape_lst = []
+    for x in np.arange(0, len(features)):
+        shape_lst.append(shape(features[x]['geometry']))
```
Comment (drewbo) [inline label_maker/main.py:30]: @martham93 can you flatten this into one list comprehension `shapes = [shape(geom) for ...]`

File: label_maker/main.py [hunk 2]
```diff
@@ -20,9 +23,14 @@
...
+
+    bounds = unary_union(shape_lst).bounds
+    return bounds
```
Comment (drewbo) [inline label_maker/main.py:33]: Let's return the bounds directly instead of making a variable

File: label_maker/main.py (hunk 1 above)
Comment (martham93) [reply label_maker/main.py]: resolved thank you

**Comment by martham93** [comment]
thanks for the style suggestion, those should all be addressed now and I added a test.  cc @drewbo I think the way I implemented the new test in a separate file that is pretty repetitive to what's in [`integration/test_classification_labels_geojson.py`](https://github.com/developmentseed/label-maker/blob/master/test/integration/test_classification_labels_geojson.py). I don't know if it would be better to add the test for the linestring geojson into the already existing polygon geojson test?

**Pull request merged**